src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -217,6 +217,8 @@ internal string GetDebuggerDisplay()
                                 }
                                 else
                                 {
+                                    Binder.ReportDiagnosticsIfObsoleteInternal(diagnostics, importedType, usingDirective.Name, importedType.ContainingSymbol, BinderFlags.None);
```

Comment:
The proposed behavior is consistent with VB. I think if we want to change it, we might want to change how VB behaves too (this has been around for a while AFAIK).